README.md

```diff
@@ -27,18 +27,53 @@ Another moving part is the new-ish Annotations 2.0 which uses Leaflet.Illustrate
 
 ## Quick install
 
-We provide an install script for Amazon's Cloud9 service, which provides standard Linux virtual machines with a web-based editor: https://aws.amazon.com/cloud9/
+We provide an install script for Amazon's Cloud9 service, which provides standard Linux virtual machines with a web-based editor: https://c9.io/login
+
 
 To use it:
 
+### Installation (do this only once):
+
 1. Click `Create new workspace`
+
 2. Name your workspace and enter `https://github.com/USERNAME/mapknitter` (where `USERNAME` is your GitHub username) under `Clone from Git or Mercurial URL`
 3. Choose `Ruby` template
+
 4. Click `Create workspace`
-5. Once it loads, in the `bash` console at the bottom of the screen, type `source install_cloud.sh`
-6. Press `Control-x` when prompted for a message.
-7. Wait for the installation to run, and when it's complete, press the `Run` button at the top of the page.
-8. Open the URL which pops up to see MapKnitter booted up. Great work!
+
+5. Once it loads in the `bash` console at the bottom of the screen, pull in the latest `main` branch (note: AWS only lets us track `master` by default. Since we're deprecating in a month, quick workaround):
+
+```Bash
+$ git fetch --all
+
+$ git checkout remotes/origin/main
+
+$ git checkout -b main
+
+$ git remote add upstream https://github.com/publiclab/mapknitter.git 
+
```

Comment:
```suggestion
git remote update
```

Reply:
@alaxalves does you remote not update unless you do this? I don't think i've ever run `$ git remote update`

Reply:
I don't get the branches "recognized"/fetched unless I `git remote update`. If I don't do this I can't checkout to another existing branch for instance. And if we have another mirror already set, such as origin it'll update it too as @gauravano said. But if we restrictly use only `main` branch - for this case I say - then I think `git remote update` really is not necessary. Your call ;)

Reply:
Ok cool I saw @gauravano agreed above so adding it!